wait-for-node-manager.sh

```diff
@@ -28,7 +28,7 @@ until [ $found = 2 ]; do
   else
     echo "Can't connect, keep trying"
   fi
-  sleep 2
+  sleep 30
```

Comment:
Oops, I think I wasn't clear. The sleep was good at 2s. 

What I meant was that in case the manager is not up in the first 30 seconds, the middleware should assume that the manager has failed to start, quit pinging, and exit. 

So with a sleep delay of 2s, you would need to add a check where if the loop has run 15 times, it should quit.

Reply:
ok